src/ContainerCheckpointModal.jsx

```diff
@@ -25,7 +25,7 @@ const ContainerCheckpointModal = ({ containerWillCheckpoint, onAddNotification }
             tcpEstablished,
         })
                 .catch(ex => {
-                    const error = cockpit.format(_("Failed to checkpoint container $0"), containerWillCheckpoint.Names);
+                    const error = cockpit.format(_("Failed to checkpoint container $0"), containerWillCheckpoint.Name);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1350-20230712-152457-eee84e1f-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)